controller/lib/core/control.h

```diff
@@ -12,13 +12,12 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-#ifndef PID_H
-#define PID_H
```

Comment:
Change comment at end of file.

Reply:
813e7f7e1905e59ab51f432c3be961de7f2e652d